conda_recipes/blender-plugin-bundle/recipe/build.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/sh
+set -xeuo pipefail
+
+INSTALL_DIR="$PREFIX/blender-plugins"
+mkdir -p $INSTALL_DIR
```

Comment:
we should quote this to avoid bash accidentally treating this as two arguments:


```suggestion
mkdir -p "$INSTALL_DIR"
```